pkg/cvo/sync_test.go

```diff
@@ -41,74 +41,77 @@ func Test_SyncWorker_apply(t *testing.T) {
 
 		check   func(*testing.T, []action)
 		wantErr bool
-	}{{
```

Comment:
Note to Reviewers: Although it looks like a lot of this test has changed, the only non-formatting change is the change in number of `Actions`.

Reply:
easy way to help review with that sort of thing is to isolate the no-op format changes in a separate commit, so folks looking at the full PR can focus on the semantic changes.

Reply:
Moved it into a separate commit